docs/DafnyRef/Types.md

```diff
@@ -1290,10 +1326,14 @@ modifier may be omitted.
 
 ## 12.3. Method Declarations {#sec-method-declarations}
 ````grammar
-MethodDecl(isGhost, allowConstructor) =
-  MethodKeyword { Attribute } [ MethodName ]
-  ( MethodSignature(isGhost)  | SignatureEllipsis_ )
-  MethodSpec [ BlockStmt ]
+MethodDecl(isGhost, allowConstructors, isWithinAbstractModule) =
+  MethodKeyword_ { Attribute } [ MethodFunctionName ]
+  ( MethodSignature_(isGhost, isCoinductive: true iff this is a least
```

Comment:
The name `isCoinductive` is probably from the first implementation of these, which only supported greatest fixpoints. A name like `extreme` (that is, either `least` or `greatest`) is better.

Reply:
OK